docker-compose.prod.yml

```diff
@@ -0,0 +1,134 @@
+#mysql container
+db:
+    restart: always
+    image: mysql:5.6
+    ports:
+        - "3306:3306"
+    environment:
+        MYSQL_ROOT_PASSWORD: changeme
+        #the values below should match the credentials used in prod.env
+        MYSQL_USER: nbsap
+        MYSQL_PASSWORD: nbsap
+        MYSQL_DATABASE: nbsap
+
+#per country container
+ch:
+    image:nbsap_web
+    volumes:
+        - ./instance_ch/:/nbsap/instance
+    ports:
+        - "8000:7012"
+    links:
+        - db
+    env_file:
+        - ./env_files/ch.env
```

Comment:
Which .env?

instance_* overrides the container's instance, why should it contain any Docker environment variables?

What do you mean by country setup? An env file?